### ISSUE TITLE
Issue with nuget package 1.0.8.8

### ISSUE THREAD

**Issue by code2learnit** [Cinchoo/ChoETL#35]
Hi
I'm trying to install nuget package 1.0.8.8, but it seems that it cannot find System.ComponentModel.DataAnnotations.4.0.0

ChoETL 1.0.8.7 was not asking for a specific version

```
    <frameworkAssemblies>
      ..
      <frameworkAssembly assemblyName="System.ComponentModel.DataAnnotations" targetFramework="" />
      ..
    </frameworkAssemblies>
```

whereas 1.0.8.8 does require 4.0.0 (at least for .Net Framework 4.5

 ```
   <dependencies>
      <group targetFramework=".NETFramework4.5">
        ...
        <dependency id="System.ComponentModel.DataAnnotations" version="4.0.0" />
        ....
      </group>
```

**Comment by Cinchoo** [comment]
Thanks for reporting it. Corrected, pushed new package 1.0.8.9. (Unlisted 1.0.8.8).

**Comment by code2learnit** [comment]
Sorry, I miss to close it
thanks
stefano
